src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishASingleFileApp.cs

```diff
@@ -278,6 +279,27 @@ public void No_runtime_files_6_0()
                 .OnlyHaveFiles(expectedFiles);
         }
 
+        [RequiresMSBuildVersionTheory("16.8.0")]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void It_supports_composite_r2r(bool extractAll)
+        {
+            var testProject = new TestProject()
+            {
+                Name = "SingleFileTest",
+                TargetFrameworks = "net6.0",
+                IsExe = true,
+            };
+
+            var testAsset = _testAssetsManager.CreateTestProject(testProject);
+            var publishCommand = new PublishCommand(testAsset);
+
+            publishCommand
+                .Execute(PublishSingleFile, ReadyToRun, ReadyToRunComposite, RuntimeIdentifier, extractAll ? IncludeAllContent: "")
+                .Should()
+                .Pass();
```

Comment:
Should we run the resulting binary?

Reply:
Right. Running the binary would be nice :-)